### PR TITLE
Projector.service assignment slider

### PR DESCRIPTION
**Pull request by r-peschke**
based on #238 user.GetTitleInformation

### CODE REVIEW

**Comment by ostcar** [comment]
@r-peschke I rebased the PR on master and now there are no changes left. Can you check if this is correct? Maybe I merged all you changes by mergin #238? If so, can we close this here?

**Comment by r-peschke** [comment]
> @r-peschke I rebased the PR on master and now there are no changes left. Can you check if this is correct? Maybe I merged all you changes by mergin #238? If so, can we close this here?
There is a new commit with the changes pushed to this PR.

File: internal/projector/slide/assignment.go [hunk 1]
```diff
@@ -4,14 +4,25 @@ import (
...
 type dbAssignment struct {
-	ID    int    `json:"id"`
-	Title string `json:"title"`
+	ID                   int    `json:"id"`
+	Title                string `json:"title"`
+	Phase                string `json:"phase"`
+	OpenPosts            int    `json:"open_posts"`
+	Description          string `json:"description"`
+	NumberPollCandidates bool   `json:"number_poll_candidates"`
+	CandidateIds         []int  `json:"candidate_ids"`
```
Comment (ostcar) [inline internal/projector/slide/assignment.go:20]: The `D` in `Ids` should be a capital letter: https://github.com/golang/go/wiki/CodeReviewComments#initialisms

**Review by ostcar — CHANGES_REQUESTED** [review]
Only one small change. Then you can merge this.

File: internal/projector/slide/assignment.go (hunk 1 above)
Comment (r-peschke) [reply internal/projector/slide/assignment.go]: Done

**Pull request merged**